src/components/InputItem/InputItem.module.css

```diff
@@ -1,4 +1,9 @@
 .wrap {
+    display: flex;
+    flex-direction: column;
+    justify-content: stretch;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/react-todo/pullRequest?prid=5576112)